src/tables.jl

```diff
@@ -0,0 +1,26 @@
+#-----------------------------------------------------------------------# Columns 
+const TableColumns = Columns{T} where {T<:NamedTuple}
+
+Tables.istable(::Type{<:TableColumns}) = true
+Tables.materializer(c::TableColumns) = Columns
```

Comment:
Hmmm, I think we need to define the right `Columns`, `table`, and `ndsparse` functions that call `Tables.rows(x)` on the input argument, like [here](https://github.com/JuliaComputing/IndexedTables.jl/pull/185/files#diff-17226d5590764ff7e308bfa5045dacfaR16).